vars/fcosKola.groovy

```diff
@@ -32,7 +38,7 @@ def call(params = [:]) {
             def parallel = params.get('parallel', 8);
             def extraArgs = params.get('extraArgs', "");
             try {
-                shwrap("cd ${cosaDir} && cosa kola run --parallel ${parallel} ${args} ${extraArgs}")
+                shwrap("cd ${cosaDir} && cosa kola run --build ${build} ${platformArgs} --parallel ${parallel} ${args} ${extraArgs}")
```

Comment:
```suggestion
                shwrap("cd ${cosaDir} && cosa kola run --build ${buildID} ${platformArgs} --parallel ${parallel} ${args} ${extraArgs}")
```

Reply:
or change `def buildID` to `def build` above

Reply:
Nice catch, thanks!